Tools/AP_Periph/of.cpp

```diff
@@ -0,0 +1,47 @@
+#include "AP_Periph.h"
+
+#ifdef HAL_PERIPH_ENABLE_OPTICALFLOW
+
+#include <dronecan_msgs.h>
+
+void AP_Periph_FW::of_init()
+{
+    of.init(-1);
+}
+
+void AP_Periph_FW::of_send_can()
+{
+    const float dt = of.integral_dt();
```

Comment:
I suspect this dt needs to be the dt between DroneCAN packets, not the internal dt.